Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/Cancellation/CancelProgressFlowNode.cs

```diff
@@ -17,7 +17,7 @@ internal class CancelProgressFlowNode : AbstractAnvilBase
         private readonly CancelProgressFlowNode m_Parent;
         private readonly ActiveLookupData<EntityProxyInstanceID> m_ProgressLookupData;
         private readonly ActiveLookupData<EntityProxyInstanceID> m_ParentProgressLookupData;
```

Comment:
Maybe a future consideration to make `EntityProxyInstanceID` implement `IEntityProxyInstance` or give each bucket its own data type. That would make opening up access a lot easier in the future.

Reply:
Added TODO and linked to #136